src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelInput.js

```diff
@@ -37,6 +38,10 @@ export function PanelInput({
         extraKeys.Esc = onEscape;
       }
 
+      const CodeMirror = await import("codemirror");
```

Comment:
This feels a bit gross, i wonder if there's a way to simplify this further...

perhaps use `setState` for CodeMirror or pass it down from the editor...

Reply:
I created a `getCodeMirror()` function with which I can get CodeMirror synchronously once it has been imported (and this component won't get rendered before that). Passing it down would have required passing it through some half dozen components, so I decided against that.